src/RawModel.cpp

```diff
@@ -515,10 +516,10 @@ void RawModel::CreateMaterialModels(
     }
 }
 
-int RawModel::GetNodeByName(const char *name) const
+int RawModel::GetNodeById(const long nodeId) const
```

Comment:
This is a little confusing: it takes a nodeId of one kind (FBX world), but it also returns a nodeId of another kind (Raw world). It's not a huge deal but I might rename it later.